net-libs/serf/serf-1.3.9-r2.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI="7"
 
-PYTHON_COMPAT=( python3_{8,9} )
+PYTHON_COMPAT=( python3_{8..10} )
 
 inherit python-any-r1 scons-utils toolchain-funcs flag-o-matic
 
```

Comment:
Few more points for the code below:

- I'd like to move first patch to separate line in PATCHES array, looks cleaner
- missing `die` on `sed` at line 39
- static-libs of not really neeeded should also be dropped (along with the patch)
- test are restricted, ideally we'd have a bug reference or some kind of explanation as to why it is disabled

Reply:
Grepping shows no reverse dependencies of `net-libs/serf[static-libs]`, so am dropping this.

Reply:
SCons build system without `serf-1.3.8-static-lib.patch` would build static library unconditionally.

Reply:
Yeah, I misread the patch, we can leave it as is now, as discussed you can move the build system to cmake in another PR

Reply:
```
== Running the unit tests ==
...........FF.FFF.FFF.FF.FFFFF....................................

There were 15 failures:
1) test_ssl_handshake: test/test_util.c:456: expected <0> but was <120199>
2) test_ssl_trust_rootca: test/test_util.c:456: expected <0> but was <120199>
3) test_ssl_certificate_chain_with_anchor: test/test_util.c:456: expected <0> but was <120199>
4) test_ssl_certificate_chain_all_from_server: test/test_util.c:456: expected <0> but was <120199>
5) test_ssl_no_servercert_callback_allok: test/test_util.c:456: expected <0> but was <120170>
6) test_ssl_large_response: test/test_util.c:456: expected <0> but was <120170>
7) test_ssl_large_request: test/test_util.c:456: expected <0> but was <120170>
8) test_ssl_client_certificate: test/test_util.c:456: expected <0> but was <120170>
9) test_ssl_future_server_cert: test/test_util.c:456: expected <0> but was <120199>
10) test_setup_ssltunnel: test/test_util.c:456: expected <0> but was <120170>
11) test_ssltunnel_basic_auth: test/test_context.c:2138: expected <0> but was <120170>
12) test_ssltunnel_basic_auth_server_has_keepalive_off: test/test_context.c:2138: expected <0> but was <120170>
13) test_ssltunnel_basic_auth_proxy_has_keepalive_off: test/test_context.c:2138: expected <0> but was <120170>
14) test_ssltunnel_basic_auth_proxy_close_conn_on_200resp: test/test_context.c:2138: expected <0> but was <120170>
15) test_ssltunnel_digest_auth: test/test_util.c:456: expected <0> but was <120170>

!!!FAILURES!!!
Runs: 66 Passes: 51 Fails: 15

ERROR: test(s) failed in 'test/test_all', exit code=1
scons: *** [check] Error 1
scons: building terminated because of errors.
 * ERROR: net-libs/serf-1.3.9-r2::gentoo failed (test phase):
 *   escons failed.
```

Reply:
Please report these upstream